pkgs/development/python-modules/botocore/default.nix

```diff
@@ -3,25 +3,23 @@
 , fetchPypi
 , python-dateutil
 , jmespath
-, docutils
 , urllib3
 , pytestCheckHook
 , jsonschema
 }:
 
 buildPythonPackage rec {
   pname = "botocore";
-  version = "1.29.79"; # N.B: if you change this, change boto3 and awscli to a matching version
+  version = "1.31.9"; # N.B: if you change this, change boto3 and awscli to a matching version
```

Comment:
Botocore should specify `format`.

Reply:
Thanks for catching that. I created https://github.com/NixOS/nixpkgs/pull/244878 against this branch to fix this, which can be pulled in. Or, we can feel free to ignore that for now, and I can rebase that after this branch gets merged.